frontend/src/pages/detail/BBDetail.tsx

```diff
@@ -406,26 +465,57 @@ export default function BBDetail() {
     });
   }, []);
 
+  // The navigator to switch pages
+  const navigate = useNavigate();
+
   return (
     <div>
       <Toast ref={toast} />
       <div className="grid flex justify-content-center">
-        <link
-          rel="stylesheet"
-          href="https://unpkg.com/primeflex@3.1.2/primeflex.css"
-        ></link>
-        <div className="col-11">
-          <div className="pt-2">
+        <div className="flex col-12 p-0">
+          <div className="flex col-1">
+            <Button
+              type="button"
+              label="Back"
+              icon="pi pi-arrow-left"
+              onClick={() => navigate("/buy-backs")}
+              className="p-button-sm my-auto ml-1"
+            />
+          </div>
+          <div className="pt-2 col-10">
             {isBBAddPage ? (
               <h1 className="p-component p-text-secondary text-5xl text-center text-900 color: var(--surface-800);">
-                Add Buy Back Sales
+                Add Buy Back Sale
+              </h1>
+            ) : isModifiable ? (
+              <h1 className="p-component p-text-secondary text-5xl text-center text-900 color: var(--surface-800);">
+                Modify Buy Back Sale
               </h1>
             ) : (
               <h1 className="p-component p-text-secondary text-5xl text-center text-900 color: var(--surface-800);">
-                Modify Buy Back Sales
+                Buy Back Sale Details
               </h1>
             )}
           </div>
+          <div className="flex col-1">
+            {!isBBAddPage && (
+              <Button
+                type="button"
+                label="Delete"
+                tooltip="Delete this sales reconciliation"
```

Comment:
As discussed over slack, I don't think we need tooltips. Our UI is pretty intuitive at this point, and there's really nothing else that the delete button could delete besides the sales reconciliation